email_notification/helpers.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+
+from django.db.models import Q
+
+from permissions.helpers import has_unit_permission
+from reservations.models import Reservation
+from spaces.models import Unit
+from users.models import ReservationNotification, User
+
+
+def get_staff_notification_recipients(
+    reservation: Reservation, notification_settings: List[ReservationNotification]
+) -> List[str]:
+    """
+    Get users with unit roles and notifications enabled, collect the ones that can manage relevant units,
+    have matching notification setting are not the reservation creator
+    """
+    notification_recipients: List[User] = []
+    reservation_units = reservation.reservation_unit.all()
+    units = Unit.objects.filter(reservationunit__in=reservation_units)
+
+    users = User.objects.filter(
+        Q(unit_roles__isnull=False) & ~Q(reservation_notification="NONE")
+    )
+    for user in users:
+        if (
+            has_unit_permission(user, units, "can_manage_reservations")
```

Comment:
And my above comment made me think about this one. This also restricts the service_sector_role users.

Reply:
Yep, same here. This is intentional ☺️